From: Samuel Henrique <samueloph@debian.org>
Date: Sun, 21 Sep 2025 18:58:46 +0200
Subject: [PATCH] Fix example for "continue-at"

 It stopped working after we introduced the "--no-clobber" option, to
 make the example work again we just need to explicitly override it with
 "--clobber".

 Thanks to Thomas Braun for reporting it.

 Closes: https://github.com/curl/wcurl/issues/61

Co-authored-by: Sergio Durigan Junior <github@sergiodj.net>

Backported-by: Samuel Henrique <samueloph@debian.org>
 * Modify wcurl patch to apply on curl sources by changing the location of the
   wcurl script from wcurl to scripts/wcurl.
 * Drop changes to wcurl's README file as they are not in the curl sources.
---
 docs/wcurl.md | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/docs/wcurl.md b/docs/wcurl.md
index 4111af5..1864500 100644
--- a/docs/wcurl.md
+++ b/docs/wcurl.md
@@ -124,10 +124,9 @@ Download a file passing the **--progress-bar** and **--http2** flags to curl:
 
 **wcurl --curl-options="--progress-bar --http2" example.com/filename.txt**
 
-Resume from an interrupted download (if more options are used, this needs to
-be the last one in the list):
+* Resume from an interrupted download. The options necessary to resume the download (`--clobber --continue-at -`) must be the **last** options specified in `--curl-options`. Note that the only way to resume interrupted downloads is to allow wcurl to overwrite the destination file:
 
-**wcurl --curl-options="--continue-at -" example.com/filename.txt**
+**wcurl --curl-options="--clobber --continue-at -" example.com/filename.txt**
 
 # AUTHORS
 
